src/Microsoft.Extensions.DependencyModel/DependencyContextWriter.cs

```diff
@@ -127,12 +199,75 @@ private JObject WriteTargetLibrary(Library library)
                 );
         }
 
-        private JObject WriteAssemblies(IReadOnlyList<string> assemblies)
+        private JObject WritePortableTargetLibrary(RuntimeLibrary runtimeLibrary, CompilationLibrary compilationLibrary)
```

Comment:
I think it would be good to go back and look at refactoring `RuntimeLibrary` and `CompilationLibrary` later. It seems like there's quite a bit of duplication in behavior. Not something we need to deal with now though.